src/site/content/en/blog/image-policies/index.md

```diff
@@ -0,0 +1,273 @@
+optimization: `oversized-images`, `unoptimized-lossy-images`,
+`unoptimized-lossless-images`, and`unoptimized-lossless-images-strict`.
+All are now available for [origin
+trials](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md).
+
+## Optimized image policies
+
+Feature policy is introducing a new set restrictions on images that can be
+applied with development-time enforcement. Images violating any of the
+restrictions will be rendered as placeholder images, which are easy to identify
+and fix. These policies can be specified in report-only mode where images will
+render normally without enforcement while violations are being observed via
+reports. (See [Report-only mode in the wild](#report-only-mode-in-the-wild),
+below for details.)
+
+### oversized-images
+The `oversized-images` feature policy restricts the intrinsic dimensions
+of an image in relation to its container size.
+
+When a document uses the `oversized-images` policy, any `<img>` element
+whose intrinsic resolution is more than X times larger than the container size
+in either dimension will be replaced with a placeholder image.
+
+
+#### Why?
+
+Serving images larger than what the viewing device can render&mdash;for example,
+serving desktopß images to mobile contexts, or serving high-pixel-density images
+to a low-pixel-density device&mdash;is wasting network traffic and device
+memory. Read [Serve images with correct
+dimensions](https://web.dev/serve-images-with-correct-dimensions/) and [Serve
+responsive images](https://web.dev/serve-responsive-images/) for information on
+optimizing your images.
+
+#### Examples
+
+A few examples illustrate this. The following shows the default behavior when cutting an image's display size in half.
+
+![The default resizing behavior](default-100x100.png)
+
+If I apply the following feature policy, I get a placeholder image instead.
+
+`Feature-Policy: oversized-images *(2);`
+
+![When the image is too large for the container](resize-both-dimensions.png)
+
+I get similar results if I lower only the width or the height.
+
+![Resized width](resize-width.png) ![Resized height](resized-height.png)
+
+#### How to use
+
+If you are new to feature policy, please check out [Introduction to Feature
+Policy](https://developers.google.com/web/updates/2018/06/feature-policy) for
+more details.
+
+To summarize, `oversized-images` policy can be specified through either:
+
+* `Feature-Policy` HTTP header
+* `<iframe>` `allow` attribute
+
+To declare the `oversized-images` policy, you need to provide:
+
+* The feature name, `oversized-images` (Required)
+* A list of origins (Optional)
+* The threshold values (i.e., the downscaling ratio X) for the origins, specified in
+  parenthesis (Optional)
+
+We recommend a downscaling ratio of 2.0 or lower. Consider using [responsive
+images](https://web.dev/serve-responsive-images/) with different resolutions to
+best serve images on various screen sizes, resolutions, etc..
```

Comment:
```suggestion
best serve images on various screen sizes, resolutions, etc.
```

I guess it's usually either a single period, or an ellipsis. Up to you which you prefer

Reply:
This always throws people. An abbreviation and a sentence can't share a period. Therefore there are two.